assets/blocks/course-outline/module-block/edit.js

```diff
@@ -67,15 +86,28 @@ export const EditModuleBlock = ( props ) => {
 
 	const [ isExpanded, setExpanded ] = useState( true );
 
-	const blockStyleColors = {
-		default: { background: mainColor?.color },
-		minimal: { borderColor: mainColor?.color },
-	}[ blockStyle ];
+	let blockStyleColors = {};
+	const style = className.match( /is-style-(\w+)/ );
+
+	if ( style ) {
+		blockStyleColors = {
+			default: { background: mainColor?.color },
+			minimal: { borderColor: mainColor?.color },
+		}[ style[ 1 ] ];
+	}
 
 	return (
 		<>
-			<ModuleBlockSettings { ...props } />
-			<section className={ className }>
+			<ModuleBlockSettings
+				bordered={ bordered }
+				setBordered={ ( newValue ) =>
+					setAttributes( { bordered: newValue } )
+				}
+			/>
+			<section
+				className={ classnames( className, { bordered } ) }
```

Comment:
Maybe we could concatenate the module prefix in the `bordered` class, just to prevent other classes from changing this accidentally as it's a very common name. WDYT?

Another approach used in the `BEM` is a double hyphen, like `--bordered`, identifying it's a modifier. But I think the standards we're using so far are just adding the prefix.

Reply:
Good idea! 77d07707f6501d4c12914eb7ee96ba16940a4992